relationships/synthesis/INFRA-NR_DOCKER_CONTAINER-to-EXT-SERVICE.yml

```diff
@@ -0,0 +1,31 @@
+relationships:
+  - name: nrInfraDockerContainerEXTService
```

Comment:
Hi,

I think the definition below would try to relate containers to containers.
Could you double check ?

Reply:
@andre-nr, Thanks for reviewing the PR, I have updated the PR. Please review again.